src/main/java/org/ihtsdo/snowowl/authoring/single/api/service/TaskService.java

```diff
@@ -52,32 +44,16 @@
 import org.springframework.security.core.context.SecurityContext;
 import org.springframework.security.core.context.SecurityContextHolder;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.google.common.base.Strings;
-import com.google.common.cache.CacheBuilder;
-import com.google.common.cache.CacheLoader;
-import com.google.common.cache.LoadingCache;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.Sets;
-
-import net.rcarz.jiraclient.Attachment;
-import net.rcarz.jiraclient.ChangeLog;
-import net.rcarz.jiraclient.ChangeLogEntry;
-import net.rcarz.jiraclient.ChangeLogItem;
-import net.rcarz.jiraclient.Field;
-import net.rcarz.jiraclient.Issue;
-import net.rcarz.jiraclient.IssueLink;
-import net.rcarz.jiraclient.JiraClient;
-import net.rcarz.jiraclient.JiraException;
-import net.rcarz.jiraclient.Resource;
-import net.rcarz.jiraclient.RestClient;
-import net.rcarz.jiraclient.RestException;
-import net.rcarz.jiraclient.Status;
-import net.rcarz.jiraclient.Transition;
-import net.rcarz.jiraclient.User;
-import net.sf.json.JSON;
-import net.sf.json.JSONArray;
-import net.sf.json.JSONObject;
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.jms.JMSException;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.*;
+import java.util.concurrent.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
```

Comment:
Please try to avoid committing classes with only import optimisations to keep the PR relevant.

Reply:
Forgive me, it was IntelliJ being annoying. I've reverted the import changes so that the only change inside the class is the relevant change. I've turned these annoying settings off now in the settings so this shouldn't happen again